clustermesh-apiserver/clustermesh/root.go

```diff
@@ -60,10 +60,8 @@ func NewCmd(h *hive.Hive) *cobra.Command {
 		PreRun: func(cmd *cobra.Command, args []string) {
 			// Overwrite the metrics namespace with the one specific for the ClusterMesh API Server
 			metrics.Namespace = metrics.CiliumClusterMeshAPIServerNamespace
+			option.Config.SetupLogging(h.Viper(), "clustermesh-apiserver")
```

Comment:
Could you please also add the LogDriver and LogOpt flags to the clustermesh-apiserver, for consistency?

```diff
diff --git a/clustermesh-apiserver/option/config.go b/clustermesh-apiserver/option/config.go
index 49f0b5340eb9..7308d3e44105 100644
--- a/clustermesh-apiserver/option/config.go
+++ b/clustermesh-apiserver/option/config.go
@@ -26,6 +26,8 @@ const (
 // are still accessed through the global DaemonConfig variable.
 type LegacyClusterMeshConfig struct {
 	Debug          bool
+	LogDriver      []string
+	LogOpt         map[string]string
 	CRDWaitTimeout time.Duration
 }

@@ -36,6 +38,8 @@ var DefaultLegacyClusterMeshConfig = LegacyClusterMeshConfig{

 func (def LegacyClusterMeshConfig) Flags(flags *pflag.FlagSet) {
 	flags.BoolP(option.DebugArg, "D", def.Debug, "Enable debugging mode")
+	flags.StringSlice(option.LogDriver, def.LogDriver, "Logging endpoints to use (example: syslog)")
+	flags.Var(option.NewNamedMapOptions(option.LogOpt, &option.Config.LogOpt, nil), option.LogOpt, "Log driver options (example: format=json)")
 	flags.Duration(option.CRDWaitTimeout, def.CRDWaitTimeout, "Cilium will exit if CRDs are not available within this duration upon startup")
 }
```